distuv/distribution_test.go

```diff
@@ -160,7 +168,51 @@ func checkPDFContinuous(t *testing.T, i int, x []float64, p probLogprober) {
 	}
 }
 
-func checkPDFDiscrete(t *testing.T, i int, xs []float64, p probLogprober, tol float64) {
+// checkProbCdfContinuous checks that the Prob, Rand, and Quantile are all consistent.
```

Comment:
Bear with me today. I had difficulties parsing the comment and I thought that there was another function called `checkProbCdfContinuous`. I parsed it as "A checks that B. C only checks that D. However, ... therefore _this_ checks also E." Also, what does `Rand` refer to here?